src/main/apollon-editor.ts

```diff
@@ -294,6 +312,25 @@ export class ApollonEditor {
 
     // notfiy that action was done
     this.notifyModelSubscribers();
+    this.notifyDiscreteModelSubscribers();
+  };
+
+  private notifyDiscreteModelSubscribers = () => {
+    try {
+      // if state not available -> do not emit changes
+      if (!this.store) return;
+      const model = this.model;
+      if (
+        this.store.getState().lastAction.lastAction.endsWith('END') ||
```

Comment:
Looks a little bit magic. Can you add some comments what kind of actions are filtered here?
Why do you need `lastAction` twice?